src/nvim/getchar.c

```diff
@@ -846,6 +846,8 @@ void init_default_mappings(void)
   add_map((char_u *)"<C-L> <Cmd>nohlsearch<Bar>diffupdate<CR><C-L>", NORMAL, true);
   add_map((char_u *)"<C-U> <C-G>u<C-U>", INSERT, true);
   add_map((char_u *)"<C-W> <C-G>u<C-W>", INSERT, true);
+  add_map((char_u *)"* y/\\V<C-R>\"<CR>", VISUAL, true);
+  add_map((char_u *)"# y?\\V<C-R>\"<CR>", VISUAL, true);
```

Comment:
You need to double each backslash here.
```suggestion
  add_map((char_u *)"* y/\\\\V<C-R>\"<CR>", VISUAL, true);
  add_map((char_u *)"# y?\\\\V<C-R>\"<CR>", VISUAL, true);
```